modules/ossm-security-mtls.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_user_guide/ossm-security.adoc
+
+[id="ossm-security-mtls_{context}"]
+= Enabling mutual Transport Layer Security (mTLS)
+
+Mutual Transport Layer Security (mTLS) is a protocol where two parties authenticate each other at the same time. It is the default mode of authentication in some protocols (IKE, SSH) and optional in others (TLS).
+
+MTLS can be used without changes to the application or service code. The TLS is handled entirely by the service mesh infrastructure and between the two sidecar proxies.
+
+By default, {ProductName} is set to permissive mode, where the sidecars in {ProductShortName} accept both plain-text traffic and connections that are encrypted using mTLS. If a service in your mesh is communicating with a service outside the mesh, strict mTLS could break communication between those services. Use permissive mode while you migrate your workloads to {ProductShortName}.
+
+== Enabling strict mTLS across the mesh
+
+If your workloads do not communicate with services outside your mesh and communication will not be interrupted by only accepting encrypted connections, you can enable mTLS across your mesh quickly. Set `spec.istio.global.mtls.enabled` to `true` in your ServiceMeshControlPlane resource. The operator creates the required resources.
+
+[source,yaml]
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+spec:
+  istio:
+    global:
+      mtls:
+        enabled: true
+----
+
+[id="ossm-security-mtls-sidecars-incoming-services_{context}"]
+=== Configuring sidecars for incoming connections for specific services
+
+You can also configure mTLS for individual services or namespaces by creating a policy. Change the following example to include your ServiceMeshControlPlane namespace.
```

Comment:
I think you can remove the last sentence here (same below for DestinationRule), as these can be created in any namespace in the mesh and, if named default, will serve as the default policy for that namespace.  Regarding the control plane namespace, I think it's best to leave it out of the docs, as it should be configured through mtls.enabled in the SMCP to avoid issues with control plane services (e.g. prometheus, jaeger, etc.).

Reply:
lgtm